PhysicsTools/NanoAOD/plugins/GenPartIsoProducer.cc

```diff
@@ -101,13 +101,16 @@ void GenPartIsoProducer::produce(edm::Event& iEvent, const edm::EventSetup& iSet
       thisLep.SetPtEtaPhiM(lep_dressed.Pt(), lep_dressed.Eta(), lep_dressed.Phi(), lep_dressed.M());
       this_GENiso = computeIso(thisLep, packedgenParticles, gen_fsrset, true);
       Lepts_RelIso.push_back(this_GENiso);
-    } else {
-      float this_GENiso_nolep = 0.0;
+    } else if (abs(genPart->pdgId()) == 22) {
```

Comment:
to make this module reusable in the future, can you please make this configurable, either by a pdgid or vector of them. renaming `this_GENiso_gamma` to `this_GENiso` or some such

Reply:
Done, added a new extra variable (`additionalPdgId_`) callable from the config. One variable is enough, as typically this quantity serves for leptons and gammas, no need to have an array here (which would make the computation longer and the memory consumption larger again)